tests/integration/specs/multiple-resource-links-test.js

```diff
@@ -0,0 +1,91 @@
+var get = Ember.get, set = Ember.set;
+var env;
+var responses, fakeServer;
+
+module('integration/specs/multiple-resource-links-test', {
+    setup: function() {
+        fakeServer = stubServer();
+
+        responses = {
+            posts_not_compound: {
+                data: [{
+                    type: 'posts',
+                    id: '1',
+                    title: 'Rails is Omakase',
+                    links: {
+                        author: {
+                            resource: '/authors/2',
+                            type: 'authors',
+                            id: '2'
+                        }
+                    }
+                }, {
+                    type: 'posts',
+                    id: '2',
+                    title: 'TDD Is Dead lol',
+                    links: {
+                        author: {
+                            resource: '/authors/1',
```

Comment:
this is wrong, correct resource url  for a relation would be /posts/2/author, see http://jsonapi.org/format/#document-structure-resource-relationships